Hunting Queries/MultipleDataSources/StorageAccountKeyEnumerationWithSigninandAuditlogs.yaml

```diff
@@ -0,0 +1,58 @@
+id: f19f913f-292a-41ed-9ac0-f3ea5e703d36
+name: Storage Account Key Enumeration with Signin and Audit logs
+description: |
+  'This query combines different Storage alerts with CommonSecurityLogs and StorageLogs helping analysts  triage and investigate any 
+  possible Storage related attacks faster thus reducing Mean Time To Respond'
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - SigninLogs
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AuditLogs
+tactics:
+  - InitialAccess
+  - LateralMovement
+relevantTechniques:
+  - T1586
+  - T1570
+query: |
+ SigninLogs 
+  | where ResultType == "0" 
+  | where AppDisplayName !in ("Office 365 Exchange Online", "Skype for Business Online", "Office 365 SharePoint Online")
+  | project  SuccessLogonTime = TimeGenerated,UserPrincipalName, IPAddress,SuccessAppDisplayName = AppDisplayName
+  |  join kind=inner
+  (
+  AzureActivity
+  | where OperationName has_any ("List keys", "List Storage Account Keys")
```

Comment:
Please use `OperationNameValue` instead `OperationName` which is going to be removed due to new connector.  The value also need to be changed to `tolower(OperationNameValue) endswith "listkeys/action"`

Reply:
Resubmitted with changes